app/search_builders/hyrax/admin_set_search_builder.rb

```diff
@@ -8,6 +8,7 @@ class AdminSetSearchBuilder < ::SearchBuilder
     # @param [Symbol] access one of :edit, :read, or :deposit
     def initialize(context, access)
       @access = access
+      @rows = 100
```

Comment:
I had thought about that as well but went with this approach to be consistent with the CollectionSearchBuilder.. So would are you thinking, 
https://github.com/samvera/hyrax/blob/da079af2148204f580830377bb91038fc0198108/app/services/hyrax/admin_set_service.rb#L18 :

`response = context.repository.search(builder(access, 100))` 

and then setting @rows as an argument in `AdminSetSearchBuilder.initialize`?